source/org/zfin/marker/presentation/AntibodyMarkerBean.java

```diff
@@ -26,56 +27,9 @@ public class AntibodyMarkerBean extends MarkerBean {
     private int numberOfDistinctComposedTerms ;
     private List<MarkerRelationshipPresentation> antigenGenes;
     private String abRegistryID;
```

Comment:
Can we remove this accessor  then as it is replaced by the List<String> method?

Reply:
I deleted abRegistryID as suggested.  It was referenced in antibodyMarkerData.tag, but that file is not used anywhere so I deleted it.